src/makeCalculator.js

```diff
@@ -4,7 +4,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const objCalculator = {
+    operate(callback, number) {
+      callback(number);
```

Comment:
The `operate` method uses a callback function, but it does not modify the `result` property of the calculator object. Ensure that the callback function is intended to operate on the `result` property or modify the implementation to achieve the desired effect.